protocol.html

```diff
@@ -937,7 +939,7 @@ <h3 property="schema:name">Accessibility Considerations</h3>
 
                   <p id="accessible-rich-internet-applications"><strong>Accessible Applications</strong>: To help assistive technologies to provide a consistent user interface and understanding of the objects, it is strongly encouraged to follow the <cite><a href="https://www.w3.org/TR/wai-aria-1.2/" rel="cito:citesAsAuthority">Accessible Rich Internet Applications</a></cite> [<cite><a class="bibref" href="#bib-wai-aria-1.2" title="Accessible Rich Internet Applications (WAI-ARIA) 1.2">WAI-ARIA-1.2</a></cite>] recommendations. To enable semantic navigation, styling and interactive features in context of digital publishing, it is encouraged to follow the <cite><a href="https://www.w3.org/TR/dpub-aria-1.0/" rel="cito:citesAsAuthority">Digital Publishing WAI-ARIA Module 1.0</a></cite> [<cite><a class="bibref" href="#bib-dpub-aria-1.0" title="Digital Publishing WAI-ARIA Module 1.0">DPUB-ARIA-1.0</a></cite>]. To support structured graphics such as charts, graphs, technical drawings and scientific diagrams, to assistive technologies in order improve accessibility of graphics or diagrams through detailed annotations, it is encouraged to follow the <cite><a href="https://www.w3.org/TR/graphics-aria-1.0/" rel="cito:citesAsAuthority">WAI-ARIA Graphics Module</a></cite> [<cite><a class="bibref" href="#bib-graphics-aria-1.0">GRAPHICS-ARIA-1.0</a></cite>].</p>
 
-                  <p id="authoring-tool-accessibility"><strong>Authoring Tool Accessibility</strong>: To contribute to the proliferation of Web content that is accessible to a broad range of people, it is strongly encouraged to follow the <cite><a href="https://www.w3.org/TR/ATAG/" rel="cito:citesAsAuthority">Authoring Tool Accessibility Guidelines</a></cite> [<cite><a class="bibref" href="#bib-atag20" title="Authoring Tool Accessibility Guidelines (ATAG) 2.0">ATAG20</a></cite>] in the design of authoring tools to support the production of accessible content through accessible user interfaces.</p>
+                  <p id="authoring-tool-accessibility"><strong>Authoring Tool Accessibility</strong>: To contribute to the proliferation of Web content that is accessible to a broad range of people, it is strongly encouraged to follow the <cite><a href="https://www.w3.org/TR/ATAG20/" rel="cito:citesAsAuthority">Authoring Tool Accessibility Guidelines</a></cite> [<cite><a class="bibref" href="#bib-atag20" title="Authoring Tool Accessibility Guidelines (ATAG) 2.0">ATAG20</a></cite>] in the design of authoring tools to support the production of accessible content through accessible user interfaces.</p>
```

Comment:
It seems like two changes sneaked in here, the latter should probably be a different PR?

Reply:
This change ("Update link") doesn't need to be in its own PR.. but I've pushed that commit into main just now. So, if/when we merge this PR, it'll only be the commit that's intended for this PR that'll be added to main,.. then things should magically work out, right, right?

If you have a better/simple solution, let me know. I'm no git-ninja.

Reply:
OK! It could be done prettier, I think, but I can go with that :-)